lib/crypto/configure.in

```diff
@@ -791,6 +791,9 @@ else
    SSL_FLAGS=
 fi
 
+AC_CHECK_LIB([crypto], [CRYPTO_memcmp],
+                       [SSL_FLAGS="-DHAVE_OPENSSL_CRYPTO_MEMCMP $SSL_FLAGS"], [], [])
+
```

Comment:
If multiple crypto libs are available, this test might check in another lib than the one we have selected for usage. I'll push a fix for this.